Checkout/Checkout/Source/3DSHandling/ThreeDSError.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ThreeDSError.swift
+//  Checkout
+//
+//  Created by Daven.Gomes on 02/02/2022.
```

Comment:
File header show something else, is this file imported from somewhere else?

Reply:
Yes this one is just moved from previous repo, which was missed to add on previous PRs.